### PR TITLE
initial controller layout for uframe assets

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@FBRTMaka We are currently rebasing master to the release 3, so I am going to hold off on this.  I do have a question though that I put in the Changed files.

File: ooiservices/app/uframe/assetController.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+@api.route('/assets', methods=['GET'], defaults={'id': None})
+@api.route('/assets/<int:id>', methods=['GET'])
+def get_list(id):
+    '''
+    Lists all the assets if id is none, returns a single asset if id is a valid int
+    '''
+    uframe_obj = uFrameAssetCollection()
+    response = uframe_obj.fetch(id)
+    return response
```
Comment (DanielJMaher) [inline ooiservices/app/uframe/assetController.py]: Do we want to JSONify this?

**Comment by oceanzus** [comment]
Might want to rebase with master and reissue PR??

File: ooiservices/app/uframe/assetController.py (hunk 1 above)
Comment (FBRTMaka) [inline ooiservices/app/uframe/assetController.py]: No, it’s not necessary now. (You can’t jsonify it this way yet).

You said you wanted smaller PRs...

Comment (DanielJMaher) [inline ooiservices/app/uframe/assetController.py]: That is totally fine.  I was just checking you didn't forget it if you meant to include it.

Comment (FBRTMaka) [inline ooiservices/app/uframe/assetController.py]: Cool cool, nah I purposefully left it out for now.

**Comment by FBRTMaka** [comment]
@DanielJMaher Please merge this for this morning, work with B.Smith will be blocked DUN DUN DUN

**Comment by DanielJMaher** [comment]
@FBRTMaka this has been rebased?

**Pull request merged**